src/Nethermind/Nethermind.Init/Steps/InitializeNetwork.cs

```diff
@@ -207,7 +207,9 @@ private async Task Initialize(CancellationToken cancellationToken)
             // we can't add eth67 capability as default, because it needs snap protocol for syncing (GetNodeData is
             // no longer available). Eth67 should be added if snap is enabled OR sync is finished
             _api.ProtocolsManager!.AddSupportedCapability(new Capability(Protocol.Eth, 67));
-            _api.ProtocolsManager!.AddSupportedCapability(new Capability(Protocol.Eth, 68));
```

Comment:
Should we remove it also from [here](https://github.com/NethermindEth/nethermind/blob/6e86182c88015fa9c79cd2c33bf74c331bb629b1/src/Nethermind/Nethermind.Network/ProtocolsManager.cs#L188)?

Reply:
We don't need to - we will not use eth68 without enabled capability. But on the other hand initialization of eth68 handler is not needed now, so we can do it and save some resources. Yeah, I think I will comment it out too. Good catch